MeetupPlatform.Api/Features/Meetups/LeftReplyComment/LeftReplyCommentFeature.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace MeetupPlatform.Api.Features.Meetups.LeftReplyComment;
+
+using AutoMapper;
+using MeetupPlatform.Api.Authentication.Helpers;
+using MeetupPlatform.Api.Domain.Comments;
+using MeetupPlatform.Api.Persistence.Context;
+using MeetupPlatform.Api.Seedwork.WebApi;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+[ApiSection(ApiSections.Meetups)]
+public class LeftReplyCommentFeature : FeatureBase
+{
+    private readonly ApplicationContext context;
+    private readonly IMapper mapper;
+
+    public LeftReplyCommentFeature(ApplicationContext context, IMapper mapper)
+    {
+        this.context = context;
+        this.mapper = mapper;
+    }
+
+    /// <summary>
+    /// Left reply to root comment.
+    /// </summary>
+    /// <response code="404">If needed meetup is null or needed comment doesn't exist.</response>
+    /// <response code="201">Returns the new created item.</response>
+    [HttpPost("/api/meetups/{meetupId:guid}/comments/{commentId:guid}/replies")]
+    [Authorize(Roles = Roles.PlainUser)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(typeof(CreatedReplyDto), StatusCodes.Status201Created)]
+    public async Task<IActionResult> LeftReplyComment([FromRoute] Guid meetupId, 
```

Comment:
If we started putting parameters on new lines, then the first parameter (i.e. `meetupId`) should also be put on a new line